cmd/ackdev/cmd/common.go

```diff
@@ -0,0 +1,64 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"k8s.io/client-go/util/homedir"
```

Comment:
prefer `github.com/mitchellh/go-homedir`. Reason: it's already a dependency for `aws-controllers-k8s/code-generator`, it's tiny and has no dependencies.